mods/ctf/ctf_bounties/init.lua

```diff
@@ -155,3 +155,17 @@ minetest.register_chatcommand("place_bounty", {
 		return true, "Put bounty on " .. target
 	end
 })
+
+minetest.register_chatcommand("bp", {
+         description = "See bountied player and the prize",
+         func = function(name)
+                  if bountied_player then 
```

Comment:
This will fix the LuaCheck run.
```suggestion
                  if bountied_player then
```